docs/src/modules/components/SchemaReference.tsx

```diff
@@ -441,16 +441,6 @@ function JsonSchemaNameValueDisplay({
   );
 }
 
-interface HeadingProps {
-  hash: string;
-  level: 'h2' | 'h3';
-  title: string;
-}
-
-function Heading({ hash, level: Level, title }: HeadingProps) {
-  return <SectionTitle title={title} id={hash} level={Level} />;
```

Comment:
This was wrong: https://mui.com/toolpad/studio/reference/file-schema/

<img width="1010" alt="SCR-20241227-cmgo" src="https://github.com/user-attachments/assets/a4fad990-f8f5-45f5-8d4c-7466ad387df2" />